driver-sync/src/test/functional/com/mongodb/client/AbstractExplainTest.java

```diff
@@ -80,4 +83,39 @@ public void testExplain() {
         assertTrue(explainBsonDocument.containsKey("queryPlanner"));
         assertFalse(explainBsonDocument.containsKey("executionStats"));
     }
+
+    @Test
+    public void testExplainOfAggregate() {
```

Comment:
As this is an integration test I suggest adding a simple pre 4.2 test just to ensure that explain returns a result and doesnt error.

Reply:
Yeah, good idea.  I was being lazy.

Done